src/UI/Dashboard/ParentDashboard.php

```diff
@@ -153,7 +155,11 @@ protected function renderChildDashboard($gibbonPersonID, $dateStart)
 
         $return = false;
 
-        $alert = getAlert($guid, $connection2, 002);
+        /**
+         * @var AlertLevelGateway
+         */
+        $alertLevelGateway = $container->get(AlertLevelGateway::class);
```

Comment:
This class has a container-aware interface, which auto-injects a reference to the container and enables the getContainer() method internally. This is a bit cleaner way for classes that need access to the container to use it, rather than globals.

```suggestion
        $alertLevelGateway = $this->getContainer()->get(AlertLevelGateway::class);
```